tests/unit/utils/test_fs.py

```diff
@@ -94,6 +94,14 @@ def base_path_is_symlink(path):
         ):
             self.assertFalse(contains_symlink_up_to(target_path, base_path))
 
+    def test_path_object_and_str_are_valid_arg_types(self):
+        base_path = "foo"
+        target_path = os.path.join(base_path, "bar")
+        self.assertFalse(contains_symlink_up_to(target_path, base_path))
+        self.assertFalse(
+            contains_symlink_up_to(PathInfo(target_path), PathInfo(base_path))
+        )
```

Comment:
Could also add a case for one str and one path_info, but the current one is good enough :)